src/playlist-selectors.js

```diff
@@ -113,81 +113,102 @@ export const comparePlaylistResolution = function(left, right) {
   return leftWidth - rightWidth;
 };
 
-const simpleSelector = function(master, bandwidth, width, height) {
-  let sortedPlaylists = master.playlists.slice();
-  let bandwidthPlaylists = [];
-  let bandwidthBestVariant;
-  let resolutionPlusOne;
-  let resolutionBestVariant;
-  let haveResolution;
-  let resolutionPlusOneList = [];
-  let resolutionPlusOneSmallest = [];
-  let resolutionBestVariantList = [];
-
-  stableSort(sortedPlaylists, comparePlaylistBandwidth);
+const simpleSelector = function(master, playerBandwidth, playerWidth, playerHeight) {
```

Comment:
Given that I am a big n00b it would be nice to have this function commented, at the moment I'm guessing it selects the proper rendition based on the criteria that's passed in.